### PR TITLE
update example to specify the Action Cable mount path [ci skip]

### PR DESCRIPTION
**Pull request by y-yagi**
Follow up to 8b69f1e

### CODE REVIEW

File: actioncable/README.md [hunk 1]
```diff
@@ -412,12 +412,12 @@ The above will start a cable server on port 28080.
...
 ### In app
 
-If you are using a server that supports the [Rack socket hijacking API](http://www.rubydoc.info/github/rack/rack/file/SPEC#Hijacking), Action Cable can run alongside your Rails application. For example, to listen for WebSocket requests on `/cable`, mount the server at that path:
+If you are using a server that supports the [Rack socket hijacking API](http://www.rubydoc.info/github/rack/rack/file/SPEC#Hijacking), Action Cable can run alongside your Rails application. For example, to listen for WebSocket requests on `/websocket`, specify that path to `config.action_cable.mount_path`:
 
 ```ruby
-# config/routes.rb
-Example::Application.routes.draw do
-  mount ActionCable.server => '/cable'
+# config/environments/development.rb
```
Comment (vipulnsward) [inline actioncable/README.md]: development?

File: guides/source/action_cable_overview.md [hunk 2]
```diff
@@ -549,12 +549,13 @@ You can change that in `config/database.yml` through the `pool` attribute.
...
 Action Cable can run alongside your Rails application. For example, to
-listen for WebSocket requests on `/websocket`, mount the server at that path:
+listen for WebSocket requests on `/websocket`, specify that path to
+`config.action_cable.mount_path`:
 
 ```ruby
-# config/routes.rb
-Example::Application.routes.draw do
-  mount ActionCable.server => '/cable'
+# config/environments/development.rb
```
Comment (vipulnsward) [inline guides/source/action_cable_overview.md]: Lets change the paths to `config/application.rb` and expand on it. This is not development specific.

Comment (y-yagi) [inline guides/source/action_cable_overview.md]: Thanks! I updated.

**Comment by vipulnsward** [comment]
Thanks @y-yagi !

**Comment by y-yagi** [comment]
Thanks!

**Pull request merged**